Detectors/FIT/simulation/src/Detector.cxx

```diff
@@ -313,8 +304,6 @@ Bool_t Detector::ProcessHits(FairVolume* v)
 HitType* Detector::AddHit(float x, float y, float z, float time, float energy, Int_t trackId, Int_t detId)
 {
   mHits->emplace_back(x, y, z, time, energy, trackId, detId);
-  auto stack = (o2::Data::Stack*)fMC->GetStack();
-  stack->addHit(GetDetId());
```

Comment:
These 2 lines were added 2 days ago as an important fix and they must stay. I don't understand the refactoring of not using the AddHit function anymore. This is a nice encapsulation but for performance reasons we should probably inline it.